src/manifolds/ConnectionManifold.jl

```diff
@@ -190,6 +196,12 @@ end
     ricci_curvature(M::AbstractManifold, p, B::AbstractBasis; backend::AbstractDiffBackend = diff_backend())
 
 Compute the Ricci scalar curvature of the manifold `M` at the point `p` using basis `B`.
+The curvature is computed as the trace of the Ricci curvature tensor with respect to
+the metric, that is ``R=g^{ij}_R_{ij}`` where ``R`` is the scalar Ricci curvature at `p`,
```

Comment:
What is `g^{ij}_R_{ij}` supposed to be? Currently this would yield a double subscripts error.

Reply:
Just a typo, I'm fixing it right now.

Reply:
It should be fine now.